[Avoiding Harm] Quizzes! (#28)
fhdsl/AI_for_Decision_Makers: quizzes/02g-Avoiding_Harm-ethical_process_quiz.md

{quiz, id: ethical_process, attempts: 10}

## Ethical Process Quiz

? When should ethical considerations occur according to the proposed framework?

a) Only after an AI system is already in use
b) At the beginning of during development only
c) Just before an AI system is deployed
D) Throughout the entire process of development and use

> ? What is one consideration when reflecting on AI use during the inception of an idea?

Reviewer: ```suggestion
? What is NOT a consideration when reflecting on AI use during the inception of an idea?
```

> a) If larger training datasets are needed

Reviewer: ```suggestion
a) What training datasets are a reasonable size
```

> b) How transparency will be maintained

Reviewer: ```suggestion
b) Could you start smaller and build on your idea?
```

> b) Could the project start smaller and be built up?

Reviewer: ```suggestion
b) What would the impact be if the AI system performed poorly
```

c) Whether AI is actually appropriate for the use case

> d) If harmful content could be generated

Reviewer: ```suggestion
d) Should you start bigger?
```
